Gemfile.lock

```diff
@@ -244,6 +244,7 @@ GEM
     zeitwerk (2.6.1)
 
 PLATFORMS
+  x86_64-darwin-20
```

Comment:
This was added by running bundle install in my machine, so I guess it's safe to add it.